### PR TITLE
add ref links to ipc wp

### PR DESCRIPTION
**Pull request by sshanks-kx**

### CODE REVIEW

File: docs/wp/ipc/index.md [hunk 1]
```diff
@@ -462,7 +462,7 @@ c  | 43.7764
 
 ### Broadcast 
 
-Much of the overhead of sending a message via IPC is in serializing the data before sending. It is possible to ‘async broadcast’ the same message to multiple handles using the internal `-25!` function. This will serialize the message once and send to all handles to reduce CPU and memory load. An error in publishing to any handle will result in the message not being sent to any of the handles, regardless of the handle’s position in the list.
+Much of the overhead of sending a message via IPC is in serializing the data before sending. It is possible to ‘async broadcast’ the same message to multiple handles using the internal [`-25!`](../../basics/internal.md#-25x-async-broadcast) function. This will serialize the message once and send to all handles to reduce CPU and memory load. An error in publishing to any handle will result in the message not being sent to any of the handles, regardless of the handle’s position in the list.
```
Comment (natalietanner) [inline docs/wp/ipc/index.md]: Much of the overhead of sending a message via IPC is in serializing the data before sending. It is possible to ‘async broadcast’ the same message to multiple handles using the internal [`-25!`](../../basics/internal.md#-25x-async-broadcast) function. This serializes the message once and sends to all handles to reduce CPU and memory load. An error in publishing to any handle results in the message not being sent to any of the handles, regardless of the handle’s position in the list.

File: docs/wp/ipc/index.md [hunk 2]
```diff
@@ -494,10 +494,6 @@ q)h3"d"
 
 This can be applied to a tickerplant publishing asynchronously to multiple subscribers: RDBs, chained tickerplants, or other processes performing realtime calculations. 
 
-:fontawesome-solid-code:
-[Examples](../../basics/internal.md#-25x-async-broadcast)
-
-
 ### `.z.pg` (get) and `.z.ps` (set)
 
 When the query reaches the server, it invokes a different message handler according to whether the query was sent synchronously ([`.z.pg`](../../ref/dotz.md#zpg-get) – get) or asynchronously ([`.z.ps`](../../ref/dotz.md#zps-set) – set). The return value from `.z.pg` is sent as the response message and the return value from `.z.ps` is ignored unless it is an error. If it is an error, the message is printed to the console of the process the query is being executed on. The error will not be visible to the querying process (the client).
```
Comment (natalietanner) [inline docs/wp/ipc/index.md:499]: When the query reaches the server, it invokes a different message handler according to whether the query was sent synchronously ([`.z.pg`](../../ref/dotz.md#zpg-get) – get) or asynchronously ([`.z.ps`](../../ref/dotz.md#zps-set) – set). The return value from `.z.pg` is sent as the response message and the return value from `.z.ps` is ignored unless it is an error. If it is an error, the message is printed to the console of the process the query is being executed on. The error is not visible to the querying process (the client).

File: docs/wp/ipc/index.md (hunk 1 above)
Comment (sshanks-kx) [reply docs/wp/ipc/index.md]: updated

File: docs/wp/ipc/index.md (hunk 2 above)
Comment (sshanks-kx) [reply docs/wp/ipc/index.md:499]: change made

**Pull request merged**